server/controllers/ticket/change-priority.php

```diff
@@ -50,23 +50,18 @@ public function handler() {
         $ticket = Ticket::getByTicketNumber($ticketNumber);
         $user = Controller::getLoggedUser();
 
-        if($ticket->owner && $user->id === $ticket->owner->id) {
-            $ticket->priority = $priority;
-            $ticket->unread = !$ticket->isAuthor($user);
-            $event = Ticketevent::getEvent(Ticketevent::PRIORITY_CHANGED);
-            $event->setProperties(array(
-                'authorStaff' => Controller::getLoggedUser(),
-                'content' => $ticket->priority,
-                'date' => Date::getCurrentDate()
-            ));
-            $ticket->addEvent($event);
-            $ticket->store();
-
-            Log::createLog('PRIORITY_CHANGED', $ticket->ticketNumber);
-            Response::respondSuccess();
-        } else {
-            throw new RequestException(ERRORS::NO_PERMISSION);
-        }
+        $ticket->priority = $priority;
+        $ticket->unread = !$ticket->isAuthor($user);
+        $event = Ticketevent::getEvent(Ticketevent::PRIORITY_CHANGED);
+        $event->setProperties(array(
+            'authorStaff' => Controller::getLoggedUser(),
+            'content' => $ticket->priority,
+            'date' => Date::getCurrentDate()
+        ));
+        $ticket->addEvent($event);
+        $ticket->store();
 
+        Log::createLog('PRIORITY_CHANGED', $ticket->ticketNumber);
+        Response::respondSuccess();
     }
```

Comment:
What about other paths like close/re-open/comment/change-department etc.?

Reply:
Close and comment were modified in this commit, change-department was working accordingly, and I'll check out about re-open.

Reply:
:v

Reply:
@guillegiu ^